src/Core/Users/class.user.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Meetup\Login\Core;
+
+use Meetup\Login\Core\Users\UserInterface
+
+Class User extends UserInterface{
+
+	public function __construct($email, $password){
```

Comment:
Not saying its wrong, but lets remove it for the time being. We will use a dependency injection container to create objects.